Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14 AS build
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm ci --only=production
+
+COPY . .
+
+FROM gcr.io/distroless/nodejs:14
+
+COPY --from=build /app /app
+
+EXPOSE 3000
```

Comment:
Do you envision this dockerfile to be used for development purposes on locahost or a generic one for deployment to containers?

If it's the second, should the port come from an env var? And [in the case of Heroku](https://help.heroku.com/PPBPA231/how-do-i-use-the-port-environment-variable-in-container-based-apps), you'd have to cater to its' nuances.